products/models.py

```diff
@@ -16,10 +16,13 @@ class Meta :
         db_table = 'categories'
 
 class Product(TimeStampedModel) :
+    menu        = models.ForeignKey(Menu, on_delete=models.CASCADE)
+    category    = models.ForeignKey(Category, on_delete=models.CASCADE)
     name        = models.CharField(max_length =50)
     price       = models.IntegerField()
     description = models.TextField()
     quantity    = models.IntegerField()
+    thumbnail   = models.CharField(max_length=700)
```

Comment:
image url 같은 경우 url 이라고 상세하게 컬럼명을 정해주시는게 좋습니다.
```suggestion
    thumbnail_image_url   = models.CharField(max_length=700)
```